config/initializers/devise.rb

```diff
@@ -70,7 +70,9 @@
   # given strategies, for example, `config.params_authenticatable = [:database]` will
   # enable it only for database (email + password) authentication.
   # config.params_authenticatable = true
-
+  config.jwt do |jwt|
```

Comment:
- [ ] Nice work done on your JWT setup, Kindly check, I get the below error in the screenshot when I try to set up your project. It will be good if you specify how to set up the secret credentials in your Readme.

![error](https://user-images.githubusercontent.com/48371156/187013267-06a2738a-f28d-4940-8ea4-abbe9df63ab8.png)